yarn-project/aztec-cli/README.md

```diff
@@ -156,7 +155,13 @@ This command deploys a compiled Noir contract to Aztec. It requires the path to
 Example usage:
 
 ```shell
-aztec-cli deploy path/to/contract.abi.json ...args
+aztec-cli deploy -c path/to/contract.abi.json -a ...args
+```
+
+With an Aztec example contract:
+
+```shell
+aztec-cli deploy -c ZkTokenContractAbi -a 333 0x134567890abcdef
```

Comment:
It would be great to add an additional command `example-contracts` that simply lists the abi names available. e.g. `ZkTokenContractAbi`.

Reply:
good idea 👍